packages/loot-core/src/server/accounts/transaction-rules.ts

```
@@ -759,3 +769,40 @@
+    if (trans.payee_name) {
+      let payeeId = (await getPayeeByName(trans.payee_name))?.id;
+      payeeId ??= await insertPayee({
+        name: trans.payee_name,
+      });
+
+      trans.payee = payeeId;
+    } else {
+      trans.payee = null;
+    }
+
+    delete trans.payee_name;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)